### ISSUE TITLE
Create a Loss Function that uses cross entropy

### ISSUE THREAD

**Issue by SCdoesCODE** [SCdoesCODE/neural_network#1]
Cross Entropy is most commonly used in classification models as it gives a normalized probability for all your classes. Its also easy to find resources on how to implement the backprop with cross entropy online.

**Comment by SCdoesCODE** [comment]
Introduced in commit `9cd7a19`
